context/generated/windows/Server/nanoserver/2022/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Default arguments
-ARG gitWindowsComponent='https://github.com/git-for-windows/git/releases/download/v2.50.1.windows.1/MinGit-2.50.1-64-bit.zip'
-ARG gitWindowsComponentSHA256='6f672aebe9e488a246efd6875f9197dbc0d9a40100e218acc3877cba2b206c45'
+ARG gitWindowsComponent='https://github.com/git-for-windows/git/releases/download/v2.49.0.windows.1/MinGit-2.49.0-64-bit.zip'
+ARG gitWindowsComponentSHA256='971cdee7c0feaa1e41369c46da88d1000a24e79a6f50191c820100338fb7eca5'
```

Comment:
@kilina0 may I please ask you to rebase on the current state of [development/2025.07.x](https://github.com/JetBrains/teamcity-docker-images/compare/development/2025.07.x)?

Unfortunately, a configuration file for Windows images has been filtered out and wasn't included into the merge commit, which causes the downgrade of the bundled `git` version.

I've fixed it within the target branches, thus a rebase with a config re-generation should solve the problem.